package.json

```diff
@@ -12,18 +12,26 @@
     "@wordpress/scripts": "^26.19.0",
     "chalk": "^4.1.2",
     "commander": "^9.4.1",
+    "copy-webpack-plugin": "^12.0.2",
     "fast-glob": "^3.3.2",
     "fs-extra": "^11.2.0",
     "husky": "^8.0.2",
     "lint-staged": "^13.1.0",
-    "lodash": "4.17.21"
+    "lodash": "4.17.21",
+    "npm-run-all": "^4.1.5",
+    "webpackbar": "^6.0.1"
   },
   "scripts": {
     "changelog": "./bin/plugin/cli.js changelog",
     "since": "./bin/plugin/cli.js since",
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
-    "build-plugins": "./bin/plugin/cli.js build-plugins",
+    "build-plugins": "npm-run-all build:plugin:*",
+    "build:plugin:performance-lab": "webpack --mode production --env plugin=performance-lab",
```

Comment:
See above: I don't see why we need a build process for Performance Lab. Its process works perfectly fine with only relying on `.gitattributes` for the files and directories to ignore.